### PR TITLE
finished first_reverse challenge

### PR DESCRIPTION
**Pull request by faridonfire**

### CODE REVIEW

**Comment by faridonfire** [comment]
@alberthuynh91 please code review my solution

File: solutions/first_reverse.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#Using the Ruby language, have the function FirstReverse(str) take the str parameter being passed and return the string in reversed order. 
+
+def FirstReverse(str)
+
+  # code goes here
```
Comment (alberthuynh91) [inline solutions/first_reverse.rb]: get rid of comment

File: solutions/first_reverse.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  return str.reverse
```
Comment (alberthuynh91) [inline solutions/first_reverse.rb]: no need for return

File: solutions/first_reverse.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#Using the Ruby language, have the function FirstReverse(str) take the str parameter being passed and return the string in reversed order. 
+
+def FirstReverse(str)
```
Comment (alberthuynh91) [inline solutions/first_reverse.rb]: get rid of parens

**Comment by alberthuynh91** [comment]
Changes look good, merge to master

**Comment by faridonfire** [comment]
:+1:

**Pull request merged**